### PR TITLE
Fix for the schema whitelist

### PR DESCRIPTION
**Pull request by pmzandbergen**
The identifiers used did not match with those generated by Magento. It might be good to update db_schema.xml as well, please see:

https://devdocs.magento.com/guides/v2.3/extension-dev-guide/declarative-schema/migration-commands.html#create-whitelist

### Fixed Issues (if relevant)

1. https://github.com/mageplaza/magento-2-blog/<issue_number>: Issue title
2. ...

### Manual testing scenarios

1. 
2. 

### Contribution checklist
 - [ ] Pull request has a meaningful description of its purpose
 - [ ] All commits are accompanied by meaningful commit messages
 - [ ] All new or changed code is covered with unit/integration tests (if applicable)

### CODE REVIEW

**Comment by Bang512** [comment]
Hi @pmzandbergen,

Thank you for your contribution.
We are reviewing this and going to merge this file into the master branch soon.
Have a nice day!

**Comment by sicet7** [comment]
I can confirm that this works.

I have been trying to update this module for at production site all day now, and these changes fixes my problems.

The problems i were having was there the setup:upgrade would always think there was an upgrade even though my versions in the setup_module table was the same as in the module.xml of the module.

TLDR: Running the setup:upgrade the 1st time after the composer update, the modules changes would be applied correctly, but every subsequent run of the setup:upgrade command would break saying the "duplicate foreign key" or "duplicate index".

Magento Version: 2.3.1
PHP Version: 7.2
Update: 3.0.4 -> 3.1.1

**Comment by thaimageplaza** [comment]
Thanks for your contributions! 
Your feedback helps us improve our extension and help troubleshoot for you and other customers. So I really appreciate you taking the time here. More importantly, we will surely release the function in the upcoming version.

Since there is no issue left, I want to reach out and let you know we are going to close this pull.
Please don’t hesitate to contact me if ever a problem should arise. We hope to have the pleasure of doing business with you for many years to come.

**Comment by fredden** [comment]
The issue still exists; see #260. Please can this pull request be merged and a new release created?
